src/qt/lookupspdialog.cpp

```diff
@@ -378,3 +423,28 @@ void LookupSPDialog::matchingComboBoxChanged(int idx)
 {
     updateDisplayedProperty();
 }
+
+class NumericalCmpWidgetItem : public QTableWidgetItem
+{
+public:
+    NumericalCmpWidgetItem(const QString& text) : QTableWidgetItem(text)
+    {
+    }
+
+    bool operator<(const QTableWidgetItem& other) const
```

Comment:
Better to apply `override` keyword here so it will know immediately it override base class method.